packages/suite-data/src/guide/parser.ts

```diff
@@ -71,14 +71,17 @@ export class Parser {
         // In GitBook a title is edited in a dedicated field
         // and its content is added to the markdown file as level-1 heading.
         // Find the line that starts with a single hash.
-        const title = doc
-            .toString()
-            .match(/^# (.+$)/m)?.[1]
-            .trim();
-        if (title === undefined) {
+        // Remove all backslashes in title which escape special characters in markdown syntax.
```

Comment:
At some point we'll probably want to support markdown in the title as well. This seems like a good workaround for the time being 👍